AC3.2Unit2FinalAssessment/AC3.2Unit2FinalAssessment/ControlsViewController.swift

```diff
@@ -0,0 +1,60 @@
+//
+//  ControlsViewController.swift
+//  AC3.2Unit2FinalAssessment
+//
+//  Created by Harichandan Singh on 10/6/16.
+//  Copyright © 2016 Harichandan Singh. All rights reserved.
+//
+
+import UIKit
+
+class ControlsViewController: UIViewController, UITextFieldDelegate {
+    
+    
+    //MARK: - Outlets
+    @IBOutlet weak var slider: UISlider!
+    @IBOutlet weak var stepper: UIStepper!
+    @IBOutlet weak var label: UILabel!
+    @IBOutlet weak var textField: UITextField!
+    
```

Comment:
you can use a helper function that updates all the outlet values and pass that function in slider, stepper, textFieldShouldReturn